examples/gzip/Dockerfile-service

```diff
@@ -0,0 +1,11 @@
+FROM envoyproxy/envoy-dev:latest
```

Comment:
dont use a envoy-dev - use a bare flask container - either based on debian - or see here #16325 for alpine example wip